nuxeo-core/nuxeo-core-event/src/main/java/org/nuxeo/ecm/core/work/WorkComputation.java

```diff
@@ -70,6 +73,10 @@ public void processRecord(ComputationContext context, String inputStreamName, Re
             } else {
                 new WorkHolder(work).run();
                 workIds.add(work.getId());
+                KeyValueStore kvStore = Framework.getService(KeyValueService.class)
+                                                 .getKeyValueStore(StreamWorkManager.KV_NAME);
+                kvStore.put(work.getId() + StreamWorkManager.STATE, Work.State.RUNNING.toString());
+                kvStore.setTTL(work.getId() + StreamWorkManager.STATE, 60 * 60L);
```

Comment:
Why not using directly `kvStore.put(String, String, long)`? Third arguments is the ttl.

Reply:
Good one !
Didn't see it